homeassistant/components/airthings_ble/sensor.py

```diff
@@ -185,4 +186,7 @@ def __init__(
     @property
     def native_value(self) -> StateType:
         """Return the value reported by the sensor."""
-        return self.coordinator.data.sensors[self.entity_description.key]
+        try:
+            return self.coordinator.data.sensors[self.entity_description.key]
+        except KeyError:
+            return STATE_UNAVAILABLE
```

Comment:
- Availability isn't implemented like this. Please implement the `available` property method instead.

Reply:
Done.